apps/astarte_pairing/lib/astarte_pairing/fdo/voucher_manager.ex

```diff
@@ -0,0 +1,73 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Pairing.FDO.VoucherManager do
+  alias Astarte.Pairing.Queries
+  require Logger
+
+  def save_voucher(realm_name, voucher_blob, pkey_blob) do
```

Comment:
pkey -> public? private?

```suggestion
  def save_voucher(realm_name, voucher_blob, private_key) do
```